lib/hammer_cli_foreman/compute_resource/utils.rb

```diff
@@ -8,6 +8,21 @@ def self.get_image_uuid(compute_resource_id, image_id)
       )
     end
 
+    def self.get_hostgroup_compute_resource_id(hostgroup_id)
+      hostgroup = HammerCLIForeman.record_to_common_format(
+          HammerCLIForeman.foreman_resource(:hostgroups).call(:show, 'id' => hostgroup_id)
+      )
+      compute_resource_id = hostgroup['compute_resource_id']
+      unless compute_resource_id
```

Comment:
should this be something like?
```suggestion
      if hostgroup['compute_resource_name'].present? && compute_resource_id.nil?
```
so we don't do the extra query if there is no compute resource?

Reply:
In case no ActiveSupport extension, then
```suggestion
      if hostgroup['compute_resource_name'].to_s.strip.empty? && compute_resource_id.nil?
```